src/probe/route/get-probes.ts

```diff
@@ -37,5 +37,5 @@ const handle = async (ctx: ParameterizedContext<DefaultState, DefaultContext & R
 };
 
 export const registerGetProbesRoute = (router: Router): void => {
-	router.get('/probes', handle);
+	router.get('/probes', '/probes', handle);
```

Comment:
Why there should be name + route instead of just route?

Reply:
The paths start to be messy when parameters get involved, so the router uses the name to refer to the routes internally: 

https://github.com/jsdelivr/globalping/pull/403/files#diff-3cb4878d5e7a24b1637229f8b009ba4736558babd22cf9cbbeeb6b373f69f260R27

If the route doesn't have a name, it can't be referenced, and `_matchedRouteName`, which we use here https://github.com/jsdelivr/globalping/pull/403/files#diff-ef7c9ff1935f2d36ee4af016c62171bc466fa6b570629c0f911410e4f319d18fR5 stays `undefined`, so you can't detect the current route and build a link based on it.